index.html

```diff
@@ -2847,6 +2847,28 @@ <h2>
         </p>
     </section>
 
+    <section>
+      <h2>
+Encrypted Data in DID Documents
+      </h2>
+      <p>
+DID documents are typically publicly available. Implementers can assume that any
+encrypted data placed in a DID document will eventually be made available in clear
+text to the same audience to which the encrypted data is available.
```

Comment:
Perhaps change "can assume" and "will eventually" to "should assume" and "may eventually".  Does the text need to briefly explain why?

Reply:
This looks resolved.